test/clj/clojure/lang/out_test.clj

```diff
@@ -14,13 +15,51 @@
             (str o))))))
 
 (deftest pr-test
-  (testing "print for a :default meta type that is not an IObj"
+  (testing "pr for a :default meta type that is not an IObj"
     (let [obj (reify
                 IMeta
                 (-meta [_] (array-map :type :default)))]
       (is (=
             (with-out-str (pr obj))
-            (str obj))))))
+            (str obj)))))
+
+  (testing "pr for nil is the string 'nil'"
+    (is (=
+          (with-out-str (pr nil))
+          "nil")))
+
+  (testing "pr for a keyword"
+    (is (=
+          (with-out-str (pr (keyword "test")))
+          ":test")))
+
+  (testing "pr for a symbol"
+    (is (=
+          (with-out-str (pr (symbol "test")))
+          "test")))
+
+  (testing "pr for a seq"
+    (is (re-matches
+          #".*TestSeqable.*"
+          (with-out-str (pr (test-seqable '(1 2 3)))))))
```

Comment:
It has the @hashcode thing going on and the hashcode isn't consistent. I'm realizing that I wanted a `test-seq` not `not-seqable` here though.